components/bsx/Offer/MyOffer.vue

```diff
@@ -34,7 +34,7 @@
       <NeoTableColumn
         v-slot="props"
         field="formatPrice"
-        :label="`${$t(`offer.price`)} (${chainSymbol})`"
+        :label="`${$t(`offer.price`)} (KSM)`"
```

Comment:
:D NO

Reply:
see https://github.com/kodadot/nft-gallery/pull/6732#issuecomment-1680675979

Reply:
I saw it, but rather skip it instead of harcoding

Reply:
fixed in https://github.com/kodadot/nft-gallery/pull/6732/commits/122c07915280b2600f139e455501b7098c72633f

Reply:
awesome ❤️